iceoryx_posh/test/integrationtests/test_popo_chunk_building_blocks.cpp

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2020 by Robert Bosch GmbH. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "iceoryx_posh/internal/popo/building_blocks/chunk_distributor.hpp"
+#include "iceoryx_posh/internal/popo/building_blocks/chunk_receiver.hpp"
+#include "iceoryx_posh/internal/popo/building_blocks/chunk_sender.hpp"
+#include "iceoryx_posh/mepoo/mepoo_config.hpp"
+
+#include "test.hpp"
+
+#include <chrono>
+#include <stdlib.h>
+#include <thread>
+
+using namespace ::testing;
+using namespace iox::popo;
+using namespace iox::cxx;
+using namespace iox::mepoo;
+using namespace iox::posix;
+using ::testing::Return;
+
+struct DummySample
+{
+    uint64_t dummy{42};
+};
+
+static constexpr size_t MEMORY_SIZE = 1024 * 1024;
+uint8_t m_memory[MEMORY_SIZE];
```

Comment:
this should maybe be aligned to 64 bytes, just to be safe

Reply:
Done